NEWS.md

```diff
@@ -5,6 +5,8 @@
 
 * Use correct parent class for Oracle (#685).
 
+* The Microsoft SQL Server method for `odbcDataType()` will now return `"BIGINT"`
```

Comment:
Conventionally we usually put new news bullets at the top. No need to change here, just remember in the future.

Reply:
Heard. Had to merge NEWS upstream anyway so just bumped that entry up.🐧